src/Extension/CoreExtension.php

```diff
@@ -1655,4 +1655,29 @@ function twig_check_arrow_in_sandbox(Environment $env, $arrow, $thing, $type)
         throw new RuntimeError(sprintf('The callable passed to the "%s" %s must be a Closure in sandbox mode.', $thing, $type));
     }
 }
+
+    /**
+     * Check if object attribute regardless of the value, it should return true
+     * It should be able to handle when object is ActiveRecord and have null value for a column e.g $user.RoleId
+     *
+     * @param $object
+     * @param $attribute
+     * @return bool
+     */
+    function hasAttribute($object, $attribute)
```

Comment:
```suggestion
    private function hasAttribute($object, $attribute)
```

Reply:
making the method private is making all the exists tests fail, hence I 
<img width="1579" alt="image" src="https://github.com/user-attachments/assets/c768df9b-3b24-453a-b8ab-6d38c3f0e107">